cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/UsingTest.scala

```diff
@@ -167,7 +167,10 @@ class UsingTest extends DocumentingTest {
       important {
         p("""Using `PERIODIC COMMIT` will prevent running out of memory when importing large amounts of data.
                 |However, it will also break transactional isolation and thus it should only be used where needed.""")
-        }
+      }
+      note {
+        p("The <<query-use, `USE` clause>> cannot be used together with the `PERIODIC COMMIT` clause.")
```

Comment:
```
<<query-use, `USE` clause>>
```
Having the backticks on `USE` makes it look like the link is just the word `clause`. Would it be better to not have backticks on `USE` so the whole thing looks like a link or keep it as it is?

Reply:
As above